.github/workflows/django.yml

```diff
@@ -0,0 +1,31 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ develop ]
```

Comment:
Any specific reason why we only want this to run on pushes to a develop branch?

Reply:
I didn't write this, I thought you did!

Reply:
Not I.